lib/ios-deploy.js

```diff
@@ -100,6 +100,18 @@ class IOSDeploy {
     await this.install(app);
   }
 
+  /**
+   * Return an application object if test app has 'bundleid'.
+   * The target bundleid can be User and System apps.
+   * @param {string} bundleid The bundleId to ensure it is installed
+   * @return {boolean} Returns True if the bundleid exists in the result of 'listApplications' like:
+   * { "com.apple.Preferences":{
+   *   "UIRequiredDeviceCapabilities":["arm64"],
+   *   "UIRequiresFullScreen":true,
+   *   "CFBundleInfoDictionaryVersion":"6.0",
+   *   "Entitlements":
+   *     {"com.apple.frontboard.delete-application-snapshots":true,..
+   */
```

Comment:
I can actually speed this part up by using a different API. We can directly ask the phone if a certain bundle id is present. I will try to implement that today